src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -255,4 +400,50 @@ protected KualiDecimal getSignedAmountFromOriginEntry(OriginEntryFull originEntr
         }
     }
 
+    protected void reportAllLinesForGroupAsFailuresDueToOrphanedCashAdvanceLines(
+            BiConsumer<ConcurStandardAccountingExtractDetailLine, String> lineErrorReporter) {

```

Comment:
See my comment on your other BiConsumer question, since this BiConsumer is being used for the same purpose.